### ISSUE TITLE
storage- error req.abort is not a function

### ISSUE THREAD

**Issue by leibale** [GoogleCloudPlatform/gcloud-node#724]
```
TypeError: undefined is not a function
    at DestroyableTransform.<anonymous> (/home/leibale/workspace/images-processor/node_modules/gcloud/lib/storage/file.js:565:23)
    at DestroyableTransform.emit (events.js:129:20)
    at /home/leibale/workspace/images-processor/node_modules/gcloud/lib/storage/file.js:452:21
    at f (/home/leibale/workspace/images-processor/node_modules/gcloud/node_modules/once/once.js:17:25)
    at DestroyableTransform.<anonymous> (/home/leibale/workspace/images-processor/node_modules/gcloud/lib/storage/file.js:556:11)
    at DestroyableTransform.emit (events.js:107:17)
    at DestroyableTransform.<anonymous> (/home/leibale/workspace/images-processor/node_modules/gcloud/node_modules/stream-forward/index.js:31:26)
    at DestroyableTransform.<anonymous> (/home/leibale/workspace/images-processor/node_modules/gcloud/node_modules/stream-forward/node_modules/on-everything/index.js:12:17)
    at DestroyableTransform.obj.(anonymous function) [as emit] (/home/leibale/workspace/images-processor/node_modules/gcloud/node_modules/stream-forward/node_modules/stubs/index.js:28:10)
    at StreamCache.<anonymous> (/home/leibale/workspace/images-processor/node_modules/gcloud/node_modules/stream-forward/index.js:31:26)
```

**Comment by stephenplusplus** [comment]
Thanks for reporting!

**Comment by stephenplusplus** [comment]
Can you try un- and re-installing gcloud? Just published an update to one of our dependencies: https://github.com/stephenplusplus/retry-request/commit/99a2476641903ff35bb1fa72e68542d10f4c81cc

**Comment by stephenplusplus** [comment]
This is fixed :) Re-open if you're still seeing the issue.

**Comment by stephenplusplus** [comment]
I was wrong, this was still an issue. Will be resolved in #746.
